src/send.c

```diff
@@ -53,6 +53,20 @@ format_range(char *buffer, size_t size, unsigned start, unsigned end)
 		snprintf(buffer, size, "%u:%u", start, end);
 }
 
+static void
+format_frange(char *buffer, size_t size, float start, float end)
+{
+	/* the special value 0.0 means "open range" */
+	if (start == 0.0 && end == 0.0)
```

Comment:
You can't compare float values this way.

Reply:
you are right. I've updated the code;
sorry for the long delay to respond.